lib/facilities/ppms/v1/response.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+module Facilities
+  module PPMS
+    module V1
+      class Response < Common::Base
+        attribute :body, String
+        attribute :params, Hash
+        attribute :current_page, Integer
+        attribute :per_page, Integer
+        attribute :offset, Integer
+        attribute :total_entries, Integer
+
+        def initialize(body, params = {})
+          super()
+          self.body = body
+          self.params = params
+          self.current_page = Integer(params[:page] || 1)
```

Comment:
Does the page count start from 0 or 1? If 0, wondering if the || 1 could be || 0 instead?

Reply:
If you want to take this refactor even further, you could create a pagination class and create an instance of the pagination object in the response class. Just a suggestion that could take the cleanup a step further

Reply:
I want to get rid of the V0 side.